PostmanCollections/VTEX - Recommendations BFF API.json

```diff
@@ -0,0 +1,1079 @@
+              "id": "dbf88156-2e15-45da-b615-17cbcc070d74",
+              "name": "Accepted",
+              "originalRequest": {
+                "url": {
+                  "path": [
+                    "api",
+                    "recommend-bff",
+                    "events",
+                    "product-view",
+                    "v2"
+                  ],
+                  "host": [
+                    "{{baseUrl}}"
+                  ],
+                  "query": [
+                    {
+                      "disabled": false,
+                      "description": {
+                        "content": "(Required) VTEX account name. Must match the account name in the `x-vtex-rec-origin` header.",
+                        "type": "text/plain"
+                      },
+                      "key": "an",
+                      "value": "apiexamples"
+                    }
+                  ],
+                  "variable": []
+                },
+                "header": [
+                  {
+                    "disabled": false,
+                    "description": {
+                      "content": "(Required) Required format: `{accountName}/{sourceType}/{appOrIntegrationId}`. Must match the `an` query parameter for account name.",
+                      "type": "text/plain"
+                    },
+                    "key": "x-vtex-rec-origin",
+                    "value": "apiexamples/storefront/vtex.recommendation-shelf@2.x"
+                  },
+                  {
+                    "disabled": true,
+                    "description": {
+                      "content": "",
+                      "type": "text/plain"
+                    },
+                    "key": "user-agent",
+                    "value": "ut proident amet"
+                  },
+                  {
+                    "key": "Content-Type",
+                    "value": "application/json"
+                  }
+                ],
+                "method": "POST",
+                "body": {
+                  "mode": "raw",
+                  "raw": "{\n  \"userId\": \"user-id\",\n  \"product\": \"product-id-1\",\n  \"source\": \"WEB_DESKTOP\"\n}",
+                  "options": {
+                    "raw": {
+                      "headerFamily": "json",
+                      "language": "json"
+                    }
+                  }
+                }
+              },
+              "status": "Accepted",
+              "code": 202,
+              "header": [],
+              "cookie": []
+            }
+          ],
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "255929ec-18f1-460b-86a7-db03cb42597b",
+                "type": "text/javascript",
+                "exec": [
+                  "// Validate status 2xx \npm.test(\"[POST]::/api/recommend-bff/events/product-view/v2 - Status code is 2xx\", function () {\n   pm.response.to.be.success;\n});\n"
```

Comment:
We did a small change on our endpoints format and we moved the version to the beginning. So, all routes should look like:


/api/recommend-bff/v2/*.